StreamChatUITestsAppUITests/Robots/UserRobot.swift

```diff
@@ -232,6 +231,18 @@ extension UserRobot {
         let messageCell = messageCell(withIndex: messageCellIndex)
         return tapOnMessage(messageCell)
     }
+    
+    @discardableResult
+    func tapOnRepliedMessage(_ text: String, at messageCellIndex: Int? = 0) -> Self {
```

Comment:
Isn't this the correct naming? 🤔 Otherwise, it looks we are taping on the reply of the quoted message 
```suggestion
    func tapOnQuotedMessage(_ text: String, at messageCellIndex: Int? = 0) -> Self {
```